examples/04-signals.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$loop = React\EventLoop\Factory::create();
+
+$loop->addSignal(SIGINT, $func = function ($signal) use ($loop, &$func) {
+    echo 'Signal: ', (string)$signal, PHP_EOL;
+    $loop->removeSignal(SIGINT, $func);
+    $loop->stop();
```

Comment:
The `$loop->stop()` doesn't do any harm, but it's not really needed anymore, right?

Reply:
Nope no need for it there \o/